src/os/bluestore/BlueStore.cc

```diff
@@ -3533,6 +3535,15 @@ void *BlueStore::MempoolThread::entry()
       next_resize += resize_interval;
     }
 
+    if (next_deferred_force_submit < ceph_clock_now()) {
+      if (store->get_deferred_last_submitted() + max_defer_interval <
+	  ceph_clock_now()) {
+	store->deferred_try_submit();
+      }
+      next_deferred_force_submit = ceph_clock_now();
+      next_deferred_force_submit += max_defer_interval/3;
```

Comment:
here , i think use frequency with 1/3 of max_defer_interval is ok. a little arbitrary but i think it is reasonable, no need an extra option.

Reply:
May be left a comment here or just use 1 second interval?